### PR TITLE
Fix CI after nBPF integration

### PR DESCRIPTION
**Pull request by IvanNardi**
Add one CI job testing nBPF

### CODE REVIEW

**Comment by utoni** [comment]
There is an annoying warning in the test diff's: `[Unknown]: nDPI compiled without nBPF support: skipping rule`.

**Comment by IvanNardi** [comment]
> There is an annoying warning in the test diff's: `[Unknown]: nDPI compiled without nBPF support: skipping rule`.

There is a `NDPI_LOG_ERR(...` call if there is a custom nBPF rule but no nBPF support.
This log is quite annoying in the CI, but the CI are compiled with the debug logs enabled, so it is kind of expected...
Some solutions:
1) Lower the level of the message so it is not print by-default, even if debug logs are enabled
2) Enable nBPF in all the CI (that might be usefull anyway, but we are "hiding" the issue)
3) ...

What do you think?

**Comment by utoni** [comment]
>     1. Lower the level of the message so it is not print by-default, even if debug logs are enabled
>
>     2. Enable nBPF in all the CI (that might be usefull anyway, but we are "hiding" the issue)
> 

We could switch some CI jobs to nBPF enabled and lower the level of the message.

File: configure.ac [hunk 1]
```diff
@@ -198,6 +198,7 @@ fi
 
 MACHINE=`uname -m`
 
+NBPF_ENABLED=0
 AC_MSG_CHECKING([PF_RING nBPF])
 NBPF_HOME=${PWD}/../PF_RING/userland/nbpf
```
Comment (utoni) [inline configure.ac]: A nice2have would be to let the user decide, what the path of nbpf should be.

Comment (IvanNardi) [reply configure.ac]: I add some autotool "magic": it seems to work (the user decide the path; default as before) but could you take a look, please?

Comment (utoni) [reply configure.ac]: lgtm

**Review by utoni — APPROVED** [review]
Good job!

**Pull request merged**